### ISSUE TITLE
ESC does not close the category menu

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6513]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & firefox & Safari
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Tab on the Header Category, Brands for eg.
- Press SPACEBAR to open the menu.
- Press ESC

**Actual Result**
Nothing happens. Was expecting ESC would close the menu. 
But actually, I had to press SPACEBAR again to close the menu.
Is that intended?

**Comment by mirkosoft** [comment]
@KateChuen issue has been fixed (PR pending) but don't you think dropdown toggle should happen also for `enter` key? It might be the case user is expecting enter to open the dropdown.

**Comment by KateChuen** [comment]
Not sure, I think to open/close menu, it would be SPACEBAR, and ESC also to close the menu...
But @developpeurweb and @kacperknapik are the accessibility guys. What do you think?

**Comment by developpeurweb** [comment]
@mirkosoft there's a caveat with the `enter` key as a menu opener, in the case that parent element happens to be ALSO a link, therefore `enter` should be configurable (optional) if possible, but `spacebar` should be default.
Please read more on this in the link below and ping me if any doubts: https://cxwiki.sap.com/display/spar/Key+Interactions+for+Category+Navigation+Menus

**Comment by developpeurweb** [comment]
@mirkosoft there's a caveat with the `enter` key as a menu opener, in the case that parent element happens to be ALSO a link, therefore `enter` should be configurable (optional) if possible, but `spacebar` should be default.
Please read more on this in the link below and ping me if any doubts: https://cxwiki.sap.com/display/spar/Key+Interactions+for+Category+Navigation+Menus

**Comment by KateChuen** [comment]
Tested on dev10. 
ESC now closes the category menu.
Verified on chrome, firefox & safari.
